src/main/java/baseball/service/ComputerService.java

```diff
@@ -0,0 +1,22 @@
+package baseball.service;
+
+import baseball.model.BaseballNumber;
+import camp.nextstep.edu.missionutils.Randoms;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class ComputerService {
+    public List<Integer> generateBaseballNumber() {
+        return IntStream.generate(() -> Randoms.pickNumberInRange(1, 9))
+                .distinct()
+                .limit(3)
+                .boxed()
+                .collect(Collectors.toList());
+    }
+
+    public BaseballNumber createComputerWithRandomNumber() {
+        List<Integer> randomNumbers = generateBaseballNumber();
+        return new BaseballNumber(randomNumbers);
```

Comment:
```suggestion
        return new BaseballNumber(generateBaseballNumber());
```
변수에 담아 후처리 로직이 필요하지 않으니, 이렇게 작성해도 꽤나 깔끔할 것 같다고 생각이 드네요!

Reply:
최고최고,,!!!!! 그저 빛..